src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -66,4 +65,15 @@ public static int countNegative(int[] array) {
                 .filter(value -> value < 0)
                 .count();
     }
+
+    public static void reverse(int[] array) {
+        Preconditions.checkNotNull(array);
```

Comment:
Please, use static import for checkNotNull for consistency as other developers use this method this way. Also, the checkNotNull method accepts a second parameter - a message. Add a descriptive message for this check (i.e. "Array must not be null")